exercises/allergies/canonical-data.json

```diff
@@ -66,6 +66,31 @@
               "result": false
             }
           ]
+        },
+        {
+          "description": "allergic to strawberries but not peanuts",
```

Comment:
I am wondering if it wouldn't be enough to just modify `"allergic to eggs in addition to other stuff"`.  It seems to me that the new case is attempting to test for the same thing, and doing a better job of it apparently.  I am curious to read what others think.  Generally speaking I think this is a good improvement to the exercise. :+1:   Thanks!